src/index.js

```diff
@@ -1,4 +1,13 @@
 import './styles/style.css';
 import displayCards from './modules/itemslist.js';
+import { doet } from './modules/comment.js';
 
 displayCards();
+
+const btn = document.querySelector('main');
+btn.addEventListener('click', (e) => {
+  if (e.target !== e.currentTarget) {
+    const b = e.target.id;
+    doet(b);
+  }
+});
```

Comment:
This function is going to interfere with every element inside `main`.

Every time something is clicked beside the `comment` button it fires up an error:
![image](https://user-images.githubusercontent.com/59011105/175234887-d2193c90-643f-4404-b52c-16068b84de81.png)

You could only target the comment buttons.